src/components/receipt-dokumentkrav/ReceiptDokumentkrav.tsx

```diff
@@ -16,6 +16,7 @@ import { ReceiptDokumentkravMissingItem } from "./ReceiptDokumentkravMissingItem
 import { ReceiptDocumentsNotSending } from "../receipt-documents-not-sending/ReceiptDocumentsNotSending";
 import { ReceiptDokumentkravUploadedItem } from "./ReceiptDokumentkravUploadedItem";
 import { ReceiptUploadDocuments } from "../receipt-upload-documents/ReceiptUploadDocuments";
+import { ReadMore as readMore } from "../../components/sanity/readmore/ReadMore";
```

Comment:
Hvorfor caste navnet til komponenten til readMore? Det ser ikke ut som Readmore krasjer med noe annet?

Reply:
Jeg gjorde det på samme måte som vi hadde gjort det med timeline. Men jeg kan gjerne endre det for begge slik at de begge er likt satt opp.